packages/angular_devkit/build_angular/src/protractor/works_spec.ts

```diff
@@ -133,4 +133,22 @@ describe('Protractor Builder', () => {
 
     await run.stop();
   });
+
+  it('supports serving HTTP headers for application being tested', async () => {
+    const overrides = {
+      headers: {
+        // Very strict CSP headers that should cause the application to fail to
+        // load.
+        'Content-Security-Policy': "default-src 'none';",
+      },
+    };
+
+    const run = await architect.scheduleTarget(protractorTargetSpec, overrides);
+
+    // Assert that the application did not load successfully, meaning that the
+    // CSP headers were indeed served.
```

Comment:
can you please clarify why this would fail? is it because the "angular" and "webpack" CSP policy creation would blow up? Please leave a comment for future reads of this code.